src/main/java/org/candlepin/model/AbstractHibernateCurator.java

```diff
@@ -1080,24 +706,10 @@ public void refresh(E... entities) {
     }
 
     public E evict(E entity) {
-        this.currentSession().evict(entity);
+        this.getEntityManager().detach(entity);
         return entity;
     }
```

Comment:
This method is now the same as `AbstractHibernateCurator.detach(Entity)`. I think there are some differences between these two methods. I believe that the `AbstractHibernateCurator.detach(Entity)` method should disregard modifications to an entity by removing it from the persistence context and the  `AbstractHibernateCurator.evict(Entity)` method should remove the entity from the Hibernate first level cache.

So, I think this method should still be:
```
  public E evict(E entity) {
      this.currentSession().evict(entity);
      return entity;
  }
```